easybuild/easyconfigs/n/NGS_Automated/NGS_Automated-2.0.1.eb

```diff
@@ -0,0 +1,37 @@
+name = 'NGS_Automated'
+version = '2.0.1'
+namelower = name.lower()
+
+homepage = 'https://github.com/molgenis/NGS_Automated'
+description = """Various scripts to automate the NGS pipelines implemented in the MOLGENIS Compute framework."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+easyblock = 'Tarball'
+
+#
+# Example URL: https://github.com/molgenis/NGS_Automated/archive/1.0.1.tar.gz
+# 
+source_urls = [('http://github.com/molgenis/%s/archive/' % (name))]
+sources = [('%s.tar.gz' % (version))]
+
+checksums = ['b282cc2937098392579ced781023eacb']
+
+dependencies = [
+    ('NGS_DNA', '3.4.2'),
+    ('NGS_RNA', '3.2.4'),
+    ('hashdeep', '4.4', '', ('foss','2015b'))
+]
+
+sanity_check_paths = {
+    'files': ['bin/copyProjectDataToPrm.sh',
+              'bin/copyRawDataToPrm.sh',
+              'bin/notifications.sh',
+              'etc/sharedConfig.cfg',
+              'lib/sharedFunctions.bash',
+              'bin/startPipeline.sh'],
+    'dirs': []
+}
+
+modextrapaths = {'PATH': '.'}
```

Comment:
Deze is onjuist en kan weg. In de root van NGS_Automated zitten geen binaries en als je je binaries in een subdir stopt met de naam 'bin', dan herkent EasyBuild dat al out-of-the-box en voegt die bin subdir toe aan $PATH ;).